cmd/ext.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"github.com/lunarway/shuttle/internal/extensions"
+	"github.com/spf13/cobra"
+)
+
+func newExtCmd() *cobra.Command {
+	extManager := extensions.NewExtensionsManager("some registry")
+
+	cmd := &cobra.Command{
+		Use:  "ext",
+		Long: "helps you manage shuttle extensions",
+	}
+
+	cmd.AddCommand(
+		newExtInstallCmd(extManager),
+		newExtUpdateCmd(extManager),
+		newExtInitCmd(extManager),
+	)
+
+	return cmd
+}
+
+func newExtInstallCmd(extManager *extensions.ExtensionsManager) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "install",
+		Long: "Install ensures that extensions already known about are downloaded and available",
```

Comment:
Something is off in this text? "known about" could be "registered"?

Reply:
I'll skip the entire known about and reigsterered, as we haven't implemented that part yet. (i.e. you can't choose currently) You're only getting the registry.